pkg/kn/commands/plugin/plugin_verifier_windows.go

```diff
@@ -0,0 +1,190 @@
+// Copyright © 2019 The Knative Authors
```

Comment:
Removed all the functions which are in plugin_verifier.go which are specific to non-Windows platform

Reply:
Seems like a lot of that code could be in a common file and limit the `plugin_verifier_windows.go` with only the Windows code. Is there a reason you could not do this?